### PR TITLE
Clean up property sheets and projects

### PR DESCRIPTION
**Pull request by Pistachioman**
I tried my hand at cleaning up the props and projects. The projects now have all more or less the same optimization settings, but I have no idea if some are vital and missing, so if anyone actually has some overview of those, input would be appreciated.
There are preprocessor definitions for x86 and x64, so #232 should be fixed with this, though there are like ten definitions for x86 I collected from all projects. I didn't look at the code much, so I'm not sure if some of them are not needed.
I tested building with Visual Studio 2013, so it (should) all work. I haven't touched the projects for older versions yet, because I'm not sure if it's enough to just compare diffs and manually copy over the changes, or if there's more to it.

### CODE REVIEW

**Comment by Pistachioman** [comment]
Is there any interest in this PR or should I close it? It's been a year now...

**Comment by micove** [comment]
Sorry, for breaking your PR so many times. My gitignore PR will break the last commit too since I deleted a postbuild file.

You could split the PR into smaller PR's. The Lilypad and ASIO stuff into a new PR and maybe the last commit into a new PR, etc. A divide and conquer approach! 

I looked at it and most of the stuff in commit 4-5, 7-9 could be easily merged if properly explained in a new PR. I think commit 1-3 and 6 are the cause of the holdup.

**Comment by micove** [comment]
I took your 4th commit and put it in #773 since I want to at least have everything compiling. It still appears as your commit.

**Comment by Pistachioman** [comment]
Rebasing is usually pretty straightforward, don't worry.
I'm not sure how easy it really would be to split off some commits, because they are already using the new props etc. I'll look into it.
But the real meat of the PR are 1-3 & 6 anyway, everything else is just extra.
Any other comments?

**Comment by micove** [comment]
Yeah, I realize. I meant that extra stuff should be in a different PR since it's unrelated but still useful. That should make that faster to merge. The main purpose of the PR is cleanup projects and props therefore 1-3 & 6.

Well, I looked at it like 8-10 months ago and I liked it. I have not looked at it recently to give it a deep review and sanity check. It's kind of big so I could look at it over the weekend. I would have probably split it into 3rdparty PR, Plugins PR, Core PR, Refractoring all 3 PR, etc to make it easier to review.

**Comment by Pistachioman** [comment]
Well, none of the commits are unrelated because they either clean up something, or in the case of the lilypad fix it wouldn't even compile. But I know what you mean.
The changes might seem big but like 80% are just including different props in the projects.
Back when I did this I set it up so that you can compile all 3rd party dependencies after cleaning them up etc., but that might be broken due to rebasing.

**Comment by micove** [comment]
Oh, I just fixed a build failure of Lilypad but it was unrelated to the header clash. It compiles fine now in vs2013/vs2015 but the clash is still there.

**Comment by gregory38** [comment]
@ramapcsx2 I hope you don't mind ;) I will let @micove handle it.

**Comment by micove** [comment]
@Pistachioman 

Would you mind me just starting this PR over and just incorporating most of your changes?

It's just that over the year a lot of minor things changed and I would end up doing another PR to cleanup again. Just don't delete your branch so I can cherry pick from it.

Also you can open a separate PR for the Lilypad issue.
I checked with /showincludes
```
         DeviceEnumerator.cpp
         Note: including file: c:\users\Compiler\desktop\git\pcsx2\plugins\lilypad\InputManager.h
         Note: including file: c:\users\Compiler\desktop\git\pcsx2\plugins\lilypad\DeviceEnumerator.h
         Note: including file: c:\users\Compiler\desktop\git\pcsx2\plugins\lilypad\WindowsMessaging.h
         Note: including file: c:\users\Compiler\desktop\git\pcsx2\plugins\lilypad\DirectInput.h
         Note: including file: c:\users\Compiler\desktop\git\pcsx2\plugins\lilypad\KeyboardHook.h
         Note: including file: c:\users\Compiler\desktop\git\pcsx2\plugins\lilypad\RawInput.h
         Note: including file: c:\users\Compiler\desktop\git\pcsx2\plugins\lilypad\XInput.h
         Note: including file: c:\users\Compiler\desktop\git\pcsx2\plugins\lilypad\HidDevice.h
         Note: including file: c:\users\Compiler\desktop\git\pcsx2\plugins\lilypad\DualShock3.h
         XInput.cpp
         Note: including file: C:\Program Files (x86)\Microsoft DirectX SDK (June 2010)\include\xinput.h
         Note: including file: c:\users\Compiler\desktop\git\pcsx2\plugins\lilypad\VKey.h
         Note: including file: c:\users\Compiler\desktop\git\pcsx2\plugins\lilypad\InputManager.h
```

DeviceEnumerator.cpp picks up the one from the project using #include "XInput.h" (correct).
XInput.cpp picks the DirectX one which it needs using #include <xinput.h>.

It currently works but the name clash is a tad ugly. If really wanted this should do it:
- renaming XInput.h to XInputLily.h
- DeviceEnumerator.cpp: #include "XInput.h" to #include "XInputLily.h"
- XInput.cpp add #include "XInputLily.h"
- rename in project and filter.

Basically the same thing you did but I used a different name and added it to XInput.cpp too.

**Comment by Pistachioman** [comment]
Sure, go ahead. In my case changing the header name was actually necessary to make it build, maybe because of priority changes for include directories or something. I'll make a new PR for that then.

**Comment by refractionpcsx2** [comment]
Since the recent unified solutions is this PR still required?

**Comment by micove** [comment]
This is unrelated to that.

**Comment by refractionpcsx2** [comment]
Okay so it's still relevant then? :)

**Comment by micove** [comment]
I'm working on it in a different PR. This one can be closed if you want since I already cloned pistachio's repository.

**Comment by refractionpcsx2** [comment]
Ok, just trying to keep things clean :)
